apps/chat/src/components/Chat/ModelList.tsx

```diff
@@ -257,7 +257,8 @@ const ModelGroup = ({
         </div>
         {!notAllowExpandDescription &&
           description &&
-          description.indexOf('\n\n') !== -1 && (
+          (description.indexOf('\n\n') !== -1 ||
```

Comment:
what do you think about regex for this case? to handle for example  '/n   /n', '/n         
        /n'?